tao_compiler/mlir/xla/ral/BUILD

```diff
@@ -430,6 +430,57 @@ cc_library(
     alwayslink = 1,
 )
 
+tf_gpu_kernel_library(
+    name = "transpose_gpu_lib",
+    srcs = [
+        "context/custom_library/transpose_gpu.cu.cc",
+    ],
+    hdrs = [
+        "context/custom_library/transpose.h",
+    ],
+    includes = [
+        "",
+    ],
+    copts = if_cuda_is_configured([
+        "-DGOOGLE_CUDA=1"
+    ]),
+    deps = [
+        "@local_config_cuda//cuda:cuda_headers",
+        "//tensorflow/core/kernels:conv_2d",
+    ],
+    #alwayslink = 1,
+)
+
+cc_library(
```

Comment:
what about TF bridge side?

Reply:
done, add bridge building and unit test.